buildSrc/src/main/java/com/uber/okbuck/core/manager/TransformManager.java

```diff
@@ -120,8 +142,11 @@ private void composeBuckFile(Path cacheDir) {
             .map(configFileName -> new ExportFile().name(configFileName))
             .collect(Collectors.toList()));
 
+    List<Rule> rules = rulesBuilder.build();
     File buckFile = cacheDir.resolve(OkBuckGradlePlugin.BUCK).toFile();
-    FileUtil.writeToBuckFile(rulesBuilder.build(), buckFile);
+    Multimap<String, String> loadStatements =
+        LoadStatementsUtil.getLoadStatements(rules, okbuckExt);
+    FileUtil.writeToBuckFile(loadStatements, rulesBuilder.build(), buckFile);
```

Comment:
reuse `rules` instead of `rulesBuilder.build()`

Reply:
done